app/services/geek_joke.service.js

```diff
@@ -0,0 +1,15 @@
+const axios = require('axios');
+const config = require('../../config');
+
+const getRandomJoke = () => {
+  const options = {
+    params: {
+      format: 'json'
+    }
+  };
+  return axios.default.get(config.common.geek_joke_api.url, options);
```

Comment:
when modifying this, remember to add informational logs

Reply:
`console.info` added before send request to api